src/core/FlureeClient.ts

```diff
@@ -334,12 +334,9 @@ export class FlureeClient {
         'You must connect before transacting. Try using .connect().transact() instead'
       );
     }
-    const transactionContext = transaction['@context'];
```

Comment:
@aaj3f - I updated the interface for `UpsertStatement` to not include `ledger` nor `@context` to more closely match how you're using it in the test cases. (I found it hard to test with when I was trying to upsert the poilcy data in the new signed query to flureeHosted testcase.) 
Let me know if you think the `client.upsert` should behave more like `client.transact`.. but for now I just simplified it to more closely match a basic insert clause rather than the whole transaction.